ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/planned/TestCasePlan.java

```diff
@@ -56,43 +57,15 @@
     this.topology = Objects.requireNonNull(topology, "topology");
   }
 
-  public List<KsqlPlan> getPlan() {
-    return planNode.getPlan();
-  }
-
-  public long getTimestamp() {
-    return specNode.getTimestamp();
-  }
-
-  public Map<String, String> getConfigs() {
-    return planNode.getConfigs();
-  }
-
-  public Map<String, String> getSchemas() {
-    return specNode.getSchemas();
-  }
-
```

Comment:
Inlined these as it means:
 1. Don't need to add new accessors here when new accessors are added to either `TestCaseSpecNode` or `TestCasePlanNode`, i.e. cleaner, more stable API.
 2. Makes it easier to access `TestCaseSpecNode` or `TestCasePlanNode` from the rewriter code.